install.sh

```diff
@@ -43,6 +43,7 @@ guess_system_package_manager(){
         SYSTEM_PACKAGE_TYPE="ebuild"
         SYSTEM_PACKAGE_MANAGER_INSTALL="emerge"
         SYSTEM_PACKAGE_MANAGER_UPDATE="emerge --sync"
+        PIP_INSTALL_CMD="--user"
```

Comment:
This will work only for `emerge`-based distros. `PIP_INSTALL_CMD` must be defined for other cases as well.